#!/usr/bin/env python
PACKAGE = "zero_gravity_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("decay", double_t, 0, "Decay factor for previous value", 0.93, 0.00, 1.00)
gen.add("force_factor", double_t, 0, "Factor for the incoming force", 0.00012, 0.000, 0.001)

exit(gen.generate(PACKAGE, "zero_gravity_control", "zero_gravity_control"))
